tornado-api/src/main/java/uk/ac/manchester/tornado/api/TornadoDeviceContext.java

```diff
@@ -24,7 +24,7 @@
 
 public interface TornadoDeviceContext {
 
-    TornadoTargetDevice getDevice();
+    TornadoTargetDevice getSPIRVDevice();
```

Comment:
why this is renamed to spirvdevice? it is not common this interface?

Reply:
This was due to an initial prev. refactoring. I agree with you, as this is the generic interface, I will revert back to `getDevice`